host/4/mariner/dotnet/dotnet-isolated/dotnet8-isolated-mariner.Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build the runtime from source
-ARG HOST_VERSION=4.34.2
+ARG HOST_VERSION=4.1035.1
```

Comment:
So, for this image (which I believe is not flex), we are changing to oop host. This is intentional?

Reply:
Yes, intentional!